.k8s-image-swapper.yml

```diff
@@ -46,6 +46,8 @@ target:
       tags:
         - key: CreatedBy
           value: k8s-image-swapper
+        - key: AnotherTag
+          value: k8s-image-swapper
```

Comment:
maybe we should set a different value to catch race condition of key is different but the value is not correctly set?